pkg/images/images.go

```diff
@@ -70,6 +72,7 @@ func NewResolver(c ImageOverrideConfig) (*Resolver, error) {
 	}{
 		{ETCD, c.ETCD},
 		{KubeAPIServer, c.KubeAPIServer},
+		{KubeProxy, c.KubeProxy},
```

Comment:
nit: this should go after KubeControllerManager (the Kube images are sorted alphabetically)